app/components/orkestrator-opplysning/OrkestratorOpplysningDato.tsx

```diff
@@ -0,0 +1,51 @@
+import { DatePicker, useDatepicker } from "@navikt/ds-react";
+import { FormScope, useField } from "@rvf/remix";
+import { addYears, formatISO, subYears } from "date-fns";
+
+import { IOrkestratorBarnOpplysning } from "~/models/orkestrator-opplysning.server";
+import { formaterNorskDato } from "~/utils/dato.utils";
+import { hentOrkestratorBarnOpplysningLabel } from "~/utils/orkestrator-opplysninger.utils";
+
+interface IProps {
+  opplysning: IOrkestratorBarnOpplysning;
+  formScope: FormScope<string>;
+}
+
+export function OrkestratorOpplysningDato({ opplysning, formScope }: IProps) {
+  const field = useField(formScope);
+
+  const { datepickerProps, inputProps } = useDatepicker({
+    defaultSelected: new Date(opplysning.verdi),
+    toDate: addYears(new Date(), 100),
+    fromDate: subYears(new Date(), 100),
+    locale: "nb",
+    inputFormat: "dd.MM.yyyy",
+    onDateChange: (date) => {
+      if (date) {
+        field.clearError();
```

Comment:
Quick fix for feilmelding som henger igjen når man tømme dato og velger dato på nytt.